worlds/sm64ex/__init__.py

```diff
@@ -61,7 +61,7 @@ def set_rules(self):
 
     def create_item(self, name: str) -> Item:
         item_id = item_table[name]
-        if name == "1Up Mushroom":
+        if name == "1Up Mushroom" "Cannon Unlock SL" "Cannon Unlock THI" "Cannon Unlock SSL":
```

Comment:
intresting iv never seen the syntax like that

Reply:
it becomes ` if name == "1Up MushroomCannon Unlock SLCannon Unlock THICannon Unlock SSL":`

Reply:
But that would break the logic right? I am pretty sure its meant to be:
`if name in ("1UP mushroom", "cannon unlock SL", "")

Reply:
yes, it is broken.